handbook/engineering/README.md

```diff
@@ -241,6 +241,42 @@ The on-call developer is responsible for:
 3. Comment on the GitHub issue and at-mention the PM and anyone listed in the requester field.
 4. If `customer-` labels are applied to the user story, at-mention the [VP of Customer Success](https://fleetdm.com/handbook/customer-success#team).
 
+### Development in vulnerabilities and nvd repositories
+The [nvd](https://github.com/fleetdm/nvd) and [vulnerabilities](https://github.com/fleetdm/vulnerabilities) repositories continuously create output files that are downloaded and used by our customers' Fleet servers. 
```

Comment:
Hey @sharon-fdm and @lukeheath, can we adjust the format for the handbook responsibilities here to keep with the structure of the departmental pages? All that is needed is to move lines 244 - 258 to the product groups page as those are more of the "What" and "Why". The other three sections are perfect examples of responsibilities as they explain the "How". The heading can change to H3s to match the rest of the page and be renamed in imperative mood verb phrase. Examples here: https://fleetdm.com/handbook/company/leadership#outline-of-departmental-page-structure

Reply:
@Sampfluger88, sorry for the late response.
Could you please refer me to the place you want me to move lines 244-258?

Reply:
@Sampfluger88, I believe this section goes as one part that should remain together because lines 443-456 (originally 244-258 at the time you reviewed) explain why we need the process starting line 457.

TMWYT